src/bin/nydusd/main.rs

```diff
@@ -210,7 +210,9 @@ fn append_fs_options(app: App<'static, 'static>) -> App<'static, 'static> {
         Arg::with_name("shared-dir")
             .long("shared-dir")
             .short("s")
-            .help("Directory shared by host and guest for passthroughfs, which also enables pathroughfs mode")
+            .help("Directory shared by host and guest for passthroughfs or a \
```

Comment:
In fact there are now two types of binary for nydusd, is it possible to distinguish here depending on whether the compilation feature is `fusedev` or `virtiofs`?

Reply:
It sounds reasonable and possible. But I suppose there is a concrete plan to merge `virtiofs` and `fusedev` mode into a single nydusd,  thus making it easier to be deployed and configured,  isn't it?